<?xml version="1.0"?>
<launch>
	<arg name="_port" default="/dev/ttyUSB0"/>
	<node name="RosAria" pkg="rosaria" type="RosAria"  output="screen">
		<param name="_port" value="$(arg _port)"/>
	</node>
	<arg name="_name" default="Pioneer P3-DX"/>
	<node name="rosaria_motion_node" pkg="rosaria_motion" type="rosaria_motion_node" output="screen">
		<param name="robot_name" value="$(arg _name)"/>
	</node>
	<node name="rosaria_motion_client_node" pkg="rosaria_motion" type="rosaria_motion_client_node" output="screen">
	</node>
</launch>
